script.js

```diff
@@ -1,16 +1,37 @@
 // Write your JavaScript code here!
 
-window.addEventListener("load", function() {
+window.addEventListener("load", function () {
+    const form = document.querySelector("form");
```

Comment:
I see that you are selecting the form to add event listener. You should rather select the button and add the event listener action to it. The submission happens only when the button is clicked so try and capture the click event.

Also, use `document.getElementById(...)` to select the button instead of `querySelector`